package.json

```diff
@@ -21,18 +21,26 @@
   "bugs": {
     "url": "https://github.com/codealchemist/youtube-audio-server/issues"
   },
+  "files": ["!/temp-audio/**", "src/*", "!/src/test"],
   "homepage": "https://github.com/codealchemist/youtube-audio-server#readme",
   "dependencies": {
     "express": "^4.17.1",
     "express-no-favicons": "0.0.1",
+    "ffmetadata": "^1.6.0",
```

Comment:
Could be better if we use `fluent-ffmpeg` to write metadata, which is already available.